config/dependency-injection/loader-pass.php

```diff
@@ -53,20 +52,55 @@ public function process( ContainerBuilder $container ) {
 	private function process_definition( Definition $definition, Definition $loader_definition ) {
 		$class = $definition->getClass();
 
+		try {
+			$reflect = new ReflectionClass( $class );
+			$path    = $reflect->getFileName();
+			if ( strpos( $path, 'wordpress-seo/src/helpers' )
+				|| strpos( $path, 'wordpress-seo/src/surfaces' )
+				|| strpos( $path, 'wordpress-seo/src/actions' )
+				|| strpos( $path, 'wordpress-seo/src/builders' )
+				|| strpos( $path, 'wordpress-seo/src/config' )
+				|| strpos( $path, 'wordpress-seo/src/generators' )
+				|| strpos( $path, 'wordpress-seo/src/integrations' )
+				|| strpos( $path, 'wordpress-seo/src/logger' )
+				|| strpos( $path, 'wordpress-seo/src/loader' )
+				|| strpos( $path, 'wordpress-seo/src/memoizers' )
+				|| strpos( $path, 'wordpress-seo/src/presentations' )
+				|| strpos( $path, 'wordpress-seo/src/repositories' )
+				|| strpos( $path, 'wordpress-seo/src/services' )
+				|| strpos( $path, 'wordpress-seo/src/schema-templates' )
```

Comment:
I removed this folder in [this commit](https://github.com/Yoast/wordpress-seo/pull/20041/commits/b3c9f12349f48b5f133b708da0edc23275014367).
So I think we can remove this exception?

Alphabetical here would help spot these differences. Is there a reason they are not?